ssc/test/Test/Pos/Ssc/Gen.hs

```diff
@@ -0,0 +1,29 @@
+module Test.Pos.Ssc.Gen
+       ( genAttackTarget
+       ) where
+
+import qualified Prelude
+import           Universum
+
+import qualified Data.ByteString.Char8 as BC
+import           Hedgehog
+import qualified Hedgehog.Gen as Gen
+import qualified Hedgehog.Range as Range
+import           Pos.Security.Params (AttackTarget (..))
+import           Test.Pos.Core.Gen (genStakeholderId)
+
+
+----------------------------------------------------------------------------
+-- Pos.Ssc.Security Generators
+----------------------------------------------------------------------------
+
+genAttackTarget :: Gen AttackTarget
+genAttackTarget = do
+    netHostAddr <- genHost
+    port <- Gen.word16 (Range.constant 1 65535)
+    Gen.choice [ pure Prelude.$ NetworkAddressTarget (netHostAddr, port)
```

Comment:
Why not just `$` (from Universum)?

Generally we seem to use either Universum or Prelude, and Universum has `$` anyways.

I'd recommend dropping the Prelude import and just using `$` from Universum.